lib/iob/index.js

```diff
@@ -15,6 +15,10 @@ function generate (inputs) {
     };
 
     var iobArray = [];
+    console.error(inputs.clock);
+    if (! /(Z|[+-][0-2][0-9]:[03]0)+/.test(inputs.clock) ) {
```

Comment:
Heh, according to https://en.wikipedia.org/wiki/List_of_time_zone_abbreviations Nepal actually uses a :45 offset, so updated to allow that too without a warning.  :-)